leftwm-core/src/display_servers/xlib_display_server/xwrap/mouse.rs

```diff
@@ -1,25 +1,52 @@
 //! Xlib calls related to a mouse.
 use super::{XlibError, MOUSEMASK};
 use crate::display_servers::xlib_display_server::xwrap::BUTTONMASK;
-use crate::models::FocusBehaviour;
-use crate::utils::xkeysym_lookup::ModMask;
 use crate::XWrap;
 use std::os::raw::{c_int, c_ulong};
 use x11_dl::xlib;
 
 impl XWrap {
     /// Grabs the mouse clicks of a window.
-    pub fn grab_mouse_clicks(&self, handle: xlib::Window) {
+    pub fn grab_mouse_clicks(&self, handle: xlib::Window, is_focused: bool) {
         self.ungrab_buttons(handle);
-        self.grab_buttons(handle, xlib::Button1, self.mouse_key_mask);
-        self.grab_buttons(handle, xlib::Button1, self.mouse_key_mask | xlib::ShiftMask);
-        self.grab_buttons(handle, xlib::Button3, self.mouse_key_mask);
-        self.grab_buttons(handle, xlib::Button3, self.mouse_key_mask | xlib::ShiftMask);
+        if !is_focused {
+            self.grab_buttons(handle, xlib::Button1, xlib::AnyModifier, xlib::GrabModeSync);
```

Comment:
i love this, now I can resize unfocused windows in `ClickTo` without left-clicking them first :)